source/_integrations/google_mail.markdown

```diff
@@ -27,26 +27,26 @@ If you have already set up the correct credentials, you can do step 1 and then s
 {% details "Generate Client ID and Client Secret" %}
 
 This section explains how to generate a Client ID and Client Secret on
-[Google Developers Console](https://console.cloud.google.com/apis/library/gmail.googleapis.com?project=home-assistant-17698).
+[Google Developers Console](https://console.cloud.google.com/apis/library/gmail.googleapis.com).
 
-1. First, go to the Google Developers Console to enable [Gmail API](https://console.cloud.google.com/apis/library/gmail.googleapis.com?project=home-assistant-17698)
-2. The wizard will ask you to choose a project to manage your application. Select a project and click continue.
+1. First, go to the Google Developers Console to enable [Gmail API](https://console.cloud.google.com/apis/library/gmail.googleapis.com)
+2. The wizard will ask you to choose a project to manage your application. Select a project and select **Continue**.
 3. Verify that your Gmail API was enabled and click 'Go to credentials'
-4. Navigate to APIs & Services (left sidebar) > [Credentials](https://console.cloud.google.com/apis/credentials)
+4. Navigate to **APIs & Services** (left sidebar) > [Credentials](https://console.cloud.google.com/apis/credentials)
 5. Click on the field on the left of the screen, **OAuth Consent Screen**.
 6. Select **External** and **Create**.
-7. Set the *App Name* (the name of the application asking for consent) to anything you want, e.g., *Home Assistant*.
-8. You then need to select a *Support email*. To do this, click the drop-down box and select your email address.
-9. You finally need to complete the section: *Developer contact information*. To do this, enter your email address (the same as above is fine).
+7. Set the **App Name** (the name of the application asking for consent) to anything you want, e.g., *Home Assistant*.
+8. You then need to select a **Support email**. To do this, from the dropdown menu, select your email address.
+9. You finally need to complete the section: **Developer contact information**. To do this, enter your email address (the same as above is fine).
 10. Scroll to the bottom and click **Save and Continue**. You don't have to fill out anything else, or it may enable additional review.
```

Comment:
```suggestion
10. Scroll to the bottom and select **Save and Continue**. You don't have to fill out anything else, or it may enable additional review.
```